assume/reinforcement_learning/tensorboard_logger.py

```diff
@@ -200,7 +199,7 @@ def update_tensorboard(self):
             FROM rl_params
             WHERE episode = '{self.episode}'
             AND simulation = '{self.simulation_id}'
-            AND perform_evaluation = {self.perform_evaluation}
+            AND evaluation_mode = {self.evaluation_mode}
```

Comment:
This change requires to remove or adjust the database columns, but it is fine for me :)